contracts/schemes/GenericSchemeMultiCallFactory.sol

```diff
@@ -0,0 +1,49 @@
+pragma solidity 0.5.17;
+
+import "./GenericSchemeMultiCall.sol";
+import "./SimpleSchemeConstraints.sol";
+
+/**
+ * @title 
+ */
+contract GenericSchemeMultiCallFactory {
+    uint8 public constant CUSTOM = 0;
+    uint8 public constant FAST = 1;
+    uint8 public constant NORMAL = 2;
+    uint8 public constant SLOW = 3;
+
+    function createGenericSchemeMultiCallSimple(
+        Avatar _avatar,
+        IntVoteInterface _votingMachine,
+        uint8 _voteParamsType,
+        bytes32 _voteParamsHash,
+        address[] memory _contractsWhiteList,
+        string memory _descriptionHash
+    ) public returns(address) {
+        GenericSchemeMultiCall genericSchemeMultiCall = new GenericSchemeMultiCall();
+        address simpleSchemeConstraints;
+        if (_contractsWhiteList.length > 0) {
+            simpleSchemeConstraints = new SimpleSchemeConstraints();
+            SimpleSchemeConstraints(simpleSchemeConstraints).initialize(_contractsWhiteList, _descriptionHash);
+        }
+        
+        bytes32 voteParams;
+        if (_voteParamsType == FAST) {
+            // Fast params hash
+            voteParams = bytes32(0x1b46f925b15bc0590168247d8df7f72773ca64dec3334183b5387dd3945f7f2e);
```

Comment:
I think that we might need to check here first if the params are set in the voting machine..
if not it should set it up.